internal/upgraderesponder/upgraderesponder.go

```diff
@@ -0,0 +1,67 @@
+package upgraderesponder
+
+import (
+	"context"
+
+	"github.com/epinio/epinio/helpers/kubernetes"
+	"github.com/epinio/epinio/internal/version"
+	"github.com/go-logr/logr"
+	"github.com/longhorn/upgrade-responder/client"
+)
+
+const (
+	UpgradeResponderAddress = "https://epinio.version.rancher.io/v1/checkupgrade"
```

Comment:
That is actually fine, because the endpoint of the api is running on this subdomain!

Reply:
To test it you can do a simple POST to it, and it will return the Epinio releases basically.

```bash
curl -s https://epinio.version.rancher.io/v1/checkupgrade -d '{}'  | jq
```

Reply:
I confirm that your `curl` works.

I further confirm that the curl __does not work__ when using the url found in line 5 of the `docs/howtos/upgrade-responder.md` in the command, i.e. `curl -s https://version.rancher.io/v1/checkupgrade -d '{}'  | jq`

I believe that the documentation file should be updated.
Apologies that my first comment here was not clear enough.

Reply:
Seeing https://github.com/epinio/epinio/pull/1941/commits/deb99df6da18c761b44f344d4d5b803f34fc2ee6 I am ok with the change.